### DISCUSSION TITLE
Doris SQL，在子查询里嵌套执行查询时，偶尔执行的查询结果为空

### DISCUSSION THREAD

**Discussion by zhangmyWZDY** [apache/doris#38351] [RFC]
![1](https://github.com/user-attachments/assets/910bd516-3802-4ee5-b3fc-f4cc6e143492)
![2](https://github.com/user-attachments/assets/acb0d337-6476-49d0-a38f-0cc31aa5c2bb)
如图，Doris 是三集群 三副本分布，关联的三张表的最后更新时间是7.24号（无变更场景），为什么会出现时而执行查询结果为空的情况呢？

**Comment by ixzc** [comment]
中文问题去ask论坛提问一下
https://ask.selectdb.com/
